example-workflows/cloud-run/README.md

```diff
@@ -0,0 +1,67 @@
+# Cloud Run - GitHub Actions
+
+An example workflow that uses [GitHub Actions](https://help.github.com/en/categories/automating-your-workflow-with-github-actions) to deploy a [Hello World Node.js app](index.js) to [Cloud Run (Fully Managed)](https://cloud.google.com/run/).
+
+## Workflow
+
+For pushes to the _default_ branch, `master`, the workflow will:
+
+1. Verify the Google Cloud Platform credentials are correct.
+
+1. Build, tag, and push the image to Google Container Registry.
+
+    * The image is built using Cloud Build and pushed to Google Container Registry.
+
+    * The image is available through the following tags: `latest` and first 8 of
+    the commit SHA.
+
+1. Deploy the image to [Cloud Run](https://cloud.google.com/run/).
+
+## Setup
+
+1. Create or reuse a Github repository for the example workflow:
+
+  1. [Create a repository](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-new-repository).
+
+  1. Move into your repository directory.
+
+  1. Copy the example into the repository:
+  ```
+  cp -r <path_to>/github-actions/example-workflows/cloud-run/ ./
+  ```
+
+1. Add your Project Id to the [repository's secret][secrets], named `RUN_PROJECT`.
+
+1. Update `cloud-run.yml` with the following values:
+
+  * `RUN_REGION`: the region in which the resource will be deployed.
+
+  * `SERVICE_NAME`: your preferred name of your service and image.
+
+1. [Add the the following role to your service account][roles]:
+
+  * `Cloud Run Admin`
+  * `Cloud Build Editor`
+  * `Viewer`
```

Comment:
Which Viewer? Cloud Build? Cloud Run?

Reply:
Project Viewer

Reply:
```suggestion
  * `Viewer` (roles/viewer)
```

Also I'd add `roles/run.admin` and `roles/cloudbuild.builds.editor`; these are the role IDs that the CLI needs, and would help with any miscommunication when the role titles require disambiguation.

Reply:
I'm not quite sure why you haven't run into the Service Account user role necessity here, maybe because Cloud Build is somehow acting as an intermediary?

[https://cloud.google.com/run/docs/continuous-deployment-with-cloud-build#continuous-iam](Continuous deployment with minimal IAM permissions) from the official docs explains this role is needed to ensure you can't deploy code with privileges which exceed those of the deployer.

Reply:
I've changed the permissions based on Katie's and I conversation (https://gist.github.com/glasnt/e66f944ae4136ad410ddba41a34f5bd0). I'm not basing this PR on the Cloud Build docs because those use Cloud Build to handle all aspects of building and deployment to Cloud Run which isn't what we are doing here.